tests/postgres_tests/test_indexes.py

```diff
@@ -132,6 +134,46 @@ def test_deconstruction_no_customization(self):
         self.assertEqual(kwargs, {'fields': ['title'], 'name': 'test_title_gist'})
 
 
+@skipUnlessDBFeature('has_hash_index_support')
+class HashIndexTests(PostgreSQLTestCase):
+
+    def test_suffix(self):
+        self.assertEqual(HashIndex.suffix, 'hash')
+
+    def test_eq(self):
```

Comment:
I'm unsure if we need eq/name autogeneration tests for every subclass when those methods aren't overridden. Thoughts?

Reply:
Agreed. Will look into tidying this up.